python-template/{{cookiecutter.placeholder_repo_name}}/README.rst

```diff
@@ -25,60 +25,10 @@ project.
 Getting Started
```

Comment:
[suggestion] Replace this header with "Developing." I think it's clearer than "Getting Started" (getting started on what?)

Reply:
Yeah, I like that!

Reply:
@sarina This makes sense to me, but it would no longer be in line with the OEP, as most recently updated in https://github.com/openedx/open-edx-proposals/pull/536/files -- I think I'd prefer to go with Becca's suggestion and change the OEP as well. Does that sound good to you?

Reply:
I like the _feel_ of getting started, as it feels like an entry point. What about "Getting Started with Development" or "Developing Quickstart" or something?

Reply:
I'm down with getting started with development.

Reply:
I'm feeling torn between wanting a section that offers how to get started (as in, first steps) vs. how to develop in this repo (a more full set of instructions. The current content is mostly the former, but I'd love to have the latter fleshed out somewhere. Maybe if we can get a more uniform development interface (Makefile targets and such) then the document can grow to have additional sections on topics like linting, releasing, etc.

Does that sound right, and if so, does it change anything about the heading that links there?

Reply:
That sounds right to me. "Getting Started with Development" to me describes the content that's there, ie, first steps. It would indeed be great to have more comprehensive documentation, of course

Reply:
OK cool, let's go with that here and in the OEP.

Reply:
Matching PR: https://github.com/openedx/open-edx-proposals/pull/539